hivelvet-docs/run-installer-dev.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Since docs are built, I don't it should be served via pm2. Or do we need it for dev?

Reply:
In fact, we need to serve the application for the dev (as we did on hivelvet front-end) so I used pm2, the website will be accessed using the url **http://docs.hivelvet.test:3000/**